_services_projects/legislation-as-code.md

```diff
@@ -16,3 +16,54 @@ report: "[Exploring Machine Consumable Code - ACC](/assets/Exploring_Machine_Con
 quote:
 media: "[Embracing Innovation in Government - Global Trends 2019](https://trends.oecd-opsi.org/)"
 ---
+
+## Work to date
+
+Our rules can be viewed at <https://rules.nz>
+
+### Birth of a Child (or welcoming a child into your whanau)
+
+Encoding entitlements related to the birth of a child (or welcoming a child into your whanau) into re-usable open legislation as code. This component was then used within [Smart Start - financial help](https://smartstart.services.govt.nz/financial-help).
+
+#### Child related entitlements encoded
+
+Social Security Act 2018
+
+* Accommodation Supplement
+* Childcare Subsidy
+* Child Disability Allowance
+* Community Services Card
+* Home Help
+* Job Seeker Support
+* Orphan's Benefit
+* Sole Parent Support
+* Supported Living Payment
+* Unsupported Child
+* Young Parent Payment
+
+New Zealand Superannuation and Retirement Income Act 2001
+
+* Superannuation
+
+Income Tax Act 2007
+
+* Working for Families (various)
+* Best Start
+
+Parental Leave and Employment Protection Act 1987
+
+* Paid Parental Leave
+
+### Citizenship by Grant
+
+A complex formula from within the Citizenship Act, used to calculate if an applicant has been in the country enough days to satisfy a clause. This was encoded into OpenFisca, along with some related sections of Immigration Act.
```

Comment:
Copying comment by @gthurstonnz

> Line 59: You might want to explain what OpenFisca is? Its the first time its referenced so a simple explaination about what this and how to use it would be helpful.